salt/elife-dashboard/config/srv-app-dashboard-dev-app.cfg

```diff
@@ -0,0 +1,31 @@
+[article_scheduler]
```

Comment:
The formula shouldn't need this file (although it can be shipped in the repository of `elife-dashboard` as an example)

Reply:
👍